.github/workflows/release.yaml

```diff
@@ -53,14 +53,14 @@ jobs:
         shell: bash
         run: |
           mkdir _dist
-          cp README.md LICENSE target/release/hippofactory${{ matrix.config.extension }} _dist/
+          cp README.md LICENSE target/release/hippo${{ matrix.config.extension }} _dist/
           cd _dist
-          tar czf hippofactory-${{ env.RELEASE_VERSION }}-${{ env.RUNNER_OS }}-${{ matrix.config.arch }}.tar.gz README.md LICENSE hippofactory${{ matrix.config.extension }}
+          tar czf hippo-${{ env.RELEASE_VERSION }}-${{ env.RUNNER_OS }}-${{ matrix.config.arch }}.tar.gz README.md LICENSE hippo${{ matrix.config.extension }}
```

Comment:
I'm wondering if the tar file (and the GH artifacts) should be called `hippo-cli` instead of `hippo`